platformio.ini

```diff
@@ -16,6 +16,11 @@ default_envs = lolin_d32_pro_sdmmc_pe
 board_build.flash_mode = qio
 board_build.bootloader = dio
 board_build.partitions = custom_4mb_noota.csv
+
+; performance changes, enable these to make the esp faster (also expect higher power usage)
+board_build.f_cpu = 240000000L
+;board_build.f_flash = 80000000L
```

Comment:
The commit message claims that the flash clock is set to 80 MHz, but the line is commented here. Anyway I don't think we should override the default settings from the board configurations by default. The value from the board config should already provide a sane default.

Reply:
I'll update the commit message, I tried it with 80MHz for a short time, but it did not show any improvements.